drivers/SmartThings/zwave-siren/src/aeotec-doorbell-siren/init.lua

```diff
@@ -200,11 +200,11 @@ local function activateTamper(device)
   )
 end
 
-local function clearAlarmAndChimeStateOfSoundComponents(device)
-  for endpoint = 1, NUMBER_OF_SOUND_COMPONENTS do
-    clearAlarmAndChime(device, endpoint)
-  end
-end
+-- local function clearAlarmAndChimeStateOfSoundComponents(device)
+--   for endpoint = 1, NUMBER_OF_SOUND_COMPONENTS do
+--     clearAlarmAndChime(device, endpoint)
+--   end
+-- end
```

Comment:
nit: can we remove this entirely?

Reply:
same as above here, this code will be turned back on eventually